### ISSUE TITLE
Ошибки: не ловятся ошибки в before / after (нет обёртки в try / catch)

### ISSUE THREAD

**Issue by chestozo** [pasaran/descript#59]
Поговорили голосом:
чтобы не городить огород из try / catch надо заюзать домены и там слушать ошибки - рестартовать воркеры.
http://nodejs.org/api/domain.html

В процессе переделать #55

**Comment by glukki** [comment]
К тому же try/catch, во-первых, не поймает асинхронные ошибки/исключения, во-вторых, будет тормозить весь вложенный код.

**Comment by pasaran** [comment]
Да, домены я хотел. Но они какие-то полусырые и невнятные были.
Надо еще раз на них посмотреть.

**Comment by glukki** [comment]
На всякий случай отмечу, что домены на данный момент — чтобы «собрать вещи и уйти». Воркеру надо аккуратно закончить работу (залогировать ошибку, закрыть соединения) и умереть, пока не взорвалось что-то еще.
